ppml/trusted-machine-learning/scripts/start-LDA-on-local-native.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# source code link: https://github.com/apache/spark/blob/master/examples/src/main/scala/org/apache/spark/examples/ml/LDAExample.scala
+cd /ppml
+
+input_file="data/mllib/sample_lda_libsvm_data.txt"
```

Comment:
where is data/mllib created?

Reply:
wget is only getting a file NOT creating the directory structure for you (mkdir -p /path/to/image/), you have to do this by urself